integrations/utils.ts

```diff
@@ -127,6 +96,7 @@ export function test(
             rejectDisposal = reject
           })
 
+          if (debug) console.log(`>& ${command}`)
```

Comment:
We could do the same here as above, but this now results in the same command log, but just with a different prefix:
<img width="1220" alt="image" src="https://github.com/user-attachments/assets/d04adfc0-f183-43fc-a17d-4355fc2aba95">


I would maybe just drop this? 🤔 

```suggestion
```

Reply:
@RobinMalfait I’m confused, how did you get to the above log? Were you doing both `exec()` and `spawn()`? (and if that is the case, would you not expect both to log? 🤔)